autoload/codi.vim

```diff
@@ -631,6 +636,79 @@ function! s:codi_handle_done(bufnr, output)
   keepjumps call cursor(ret_line, ret_col)
 endfunction
 
+function! s:nvim_codi_handle_done(bufnr, output)
+  " Go to target buf
+  let s:updating = 1
+  let i = s:get_codi('interpreter')
+  let num_lines = line('$')
+
+  " Preprocess if we didn't already
```

Comment:
From here until the end of the function seems to be straight copy and paste. Can we extract this bit to a function, and call that in each of `s:codi_handle_done` and `s:nvim_codi_handle_done`?